src/modules/link-selector.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Link selector
+ * @author Deux Huit Huit
+ */
+(function ($) {
+
+	'use strict';
+
+	var scope = $('body');
+	var fakeAnchor = $('<a />');
+
+	var update = function () {
+		var currentPath = window.location.pathname;
+		scope.find('[href]').each(function () {
+			var t = $(this);
+			var pathname = '';
+			var matches = [];
+
+			fakeAnchor.prop('href', t.attr('href'));
+			pathname = fakeAnchor.prop('pathname');
+			matches = _.intersection(pathname.split('/'), currentPath.split('/'));
+
+			// Partial match
+			App.modules.notify('changeState.update', {
+				item: t,
+				state: 'link-active',
```

Comment:
The active term is ambiguous with the pseudo class active. Use link-selected instead or something less ambiguous. @dominic-blain : any proposition?

Reply:
link-current-partial
link-current-exact
?

Reply:
yep like more current-link-partial (will make is-current-link-partial) and current-link (will make is-current-link). More idea?